source/playgrounds.txt

```diff
@@ -30,12 +30,40 @@ files with the ``.mongodb`` extension as playgrounds.
 
 .. include:: /includes/admonitions/note-playgrounds.rst
 
-|vsce| splits your Playground window to display the results of your 
-Playground in |json| format in the right-side pane labeled ``Playground 
-Results.json``. If you disabled split-view, |vsce| displays the 
-Playground results in |json| format in a new tab to the right of 
-your Playground. You cannot edit the results in the results pane. 
-You can save the results to a file. 
+- If you enabled split-view, |vsce| splits your Playground window to
+  display the results of your Playground. |vsce| labels this right-side
+  pane :guilabel:`Playground Result`.
+
+- If you disabled split-view, |vsce| displays the Playground results in
+  a new tab to the right of your Playground labeled
+  :guilabel:`Playground Result`.
+
+- If |vsce| returns these results in |json| format, you may edit these
+  results. If the result returns |json|\-formatted content, |vsce|
+  displays it with |json| syntax highlighting and includes an
+  :guilabel:`Edit Document` link at the top of the document.
+
+In the Playground results tab, you can:
+
+- Save the results to a file.
+- Edit the results in the results pane or tab if you made a
```

Comment:
> Edit the results in the results pane or tab if you made a

Confusing.

`Open editable json documents returned by findOne/find operations` - something like this. You again introduce some different things like pane and tabs, which are the same thing, it just changes the position in the workspace. And in your version you say again `Edit the results` - you do not edit the playground result. You open editable documents from the playground result.

Reply:
I fixed this.